wolfcrypt/src/aes.c

```diff
@@ -3060,6 +3059,62 @@ int wc_AesSetIV(Aes* aes, const byte* iv)
     #elif defined(WOLFSSL_DEVCRYPTO_AES)
         /* implemented in wolfcrypt/src/port/devcrypt/devcrypto_aes.c */
 
+    #elif defined(WOLFSSL_LINUXKM)
+
+        #ifdef WOLFSSL_AESNI
+
+        __must_check int wc_AesEncryptDirect(Aes* aes, byte* out, const byte* in)
```

Comment:
Consider macro for `__must_check` for better portability? Also saw in memory.c `WOLFSSL_LINUXKM_SIMD_X86_IRQ_ALLOWED` section.

Reply:
It's a Linux kernel macro so I left it as-was, since it's gated behind _LINUXKM.

But yes we should have a macro defined in wolfcrypt/types.h!  Will revisit next round.

Reply:
done now.